server/utils/placements/index.ts

```diff
@@ -53,9 +54,13 @@ type SpaceBookingStatus = keyof typeof statusTextMap
 const isSpaceBooking = (placement: Cas1SpaceBooking | Cas1SpaceBookingSummary): placement is Cas1SpaceBooking =>
   Boolean((placement as Cas1SpaceBooking).otherBookingsInPremisesForCrn)
 
+const isCancelled = (placement: Cas1SpaceBooking | Cas1SpaceBookingSummary): boolean =>
+  'isCancelled' in placement ? placement.isCancelled : Boolean(placement.cancellation)
+
 export const overallStatus = (placement: Cas1SpaceBookingSummary | Cas1SpaceBooking): SpaceBookingOverallStatus => {
   const isNonArrival = isSpaceBooking(placement) ? placement.nonArrival : placement.isNonArrival
 
+  if (isCancelled(placement)) return 'cancelled'
```

Comment:
This could probably be replaced with a similar approach to the `notArrived` status below to keep things in one place (sorry, suggested change formatting is a bit whack: suggestion is to remove `isCancelled` method and simply rely on `isSpaceBooking` type guard to return the correct property, as is done with `isNonArrival`):

```suggestion
export const overallStatus = (placement: Cas1SpaceBookingSummary | Cas1SpaceBooking): SpaceBookingOverallStatus => {
  const isNonArrival = isSpaceBooking(placement) ? placement.nonArrival : placement.isNonArrival
  const isCancelled = isSpaceBooking(placement) ? placement.cancellation : placement.isCancelled

  if (isCancelled) return 'cancelled'
```

Reply:
Makes sense.